src/compiler/diagnostic.h

```diff
@@ -180,8 +185,8 @@ class NullDiagnostics : public Diagnostics {
   void end_group() {}
 
  protected:
-  void emit(Severity severity, const char* format, va_list& arguments) {}
-  void emit(Severity severity, Source::Range range, const char* format, va_list& arguments) {}
+  bool emit(Severity severity, const char* format, va_list& arguments) { return true; }
```

Comment:
Maybe add a comment to explain why returning true here is better than returning false?

Reply:
It doesn't really matter, but I agree "false" is better.

Reply:
Turns out, that we check whether we encountered ("printed") errors in asserts. So changing it back to "true".